components/date-picker/date-picker.css

```diff
@@ -108,6 +108,25 @@
   width: calc(unit * 13);
 }
 
+.dateInput {
+  flex-basis: calc(unit * 11);
+  flex-grow: 1;
+  flex-shrink: 0;
+}
+
+.timeInput {
+  position: relative;
+
+  &::before {
+    position: absolute;
+    left: calc(-1 * unit);
+
+    content: ',';
```

Comment:
Always having comma looks weird when, for example, there is no date set
![image](https://user-images.githubusercontent.com/4318513/71406234-9d7df580-2648-11ea-9d5a-e91a3931d43e.png)

Reply:
It was made in analogy to clearable range with only difference in placeholders presence

<img width="426" alt="Screenshot 2019-12-24 at 14 09 42" src="https://user-images.githubusercontent.com/40162440/71410651-2ac84680-2657-11ea-991b-7bef5286c47e.png">

But since mockup says range should behave like you suggested - I'll look into both

<img width="744" alt="Screenshot 2019-12-24 at 14 12 10" src="https://user-images.githubusercontent.com/40162440/71410713-6a8f2e00-2657-11ea-95dd-b8daa3680705.png">

Reply:
I removed comma (withTime) and dash (range) in some usecases, one of them is when both inputs are empty

<img width="450" alt="Screenshot 2019-12-27 at 12 52 08" src="https://user-images.githubusercontent.com/40162440/71517975-901f7000-28c1-11ea-8ef7-83064f49f50c.png">

I decided to not delete the second input when props are empty as these usecases are handled well enough in both range and withTime, they also give flexibility of what to pick first

Reply:
Looks much better, thanks!